src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/dashboard/dashboard.component.html

```diff
@@ -1,32 +1,39 @@
-<main aria-label="Dashboard" >
+<main aria-label="Dashboard">
   <a href="#main"
      class="sr-only">skip to content</a>
 
-  <cd-refresh-selector></cd-refresh-selector>
+  <div *ngIf="(enabledFeature$ | async)?.dashboard === false; else dashboardV3"
```

Comment:
No need to add a `<div>` here, which is an HTML/DOM structural component that could interfere with the CSS styling. You should use [`ng-container`](https://vibhas1892.medium.com/difference-between-ng-template-ng-container-and-ng-content-a1d264619655) instead.